inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -244,7 +244,8 @@ def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             timeout_time = current_app.config[config_key]
-            return timeout(timeout_time)(func)(*args, **kwargs)
+            use_signals = current_app.config.get('USE_SIGNALS_ON_TIMEOUT', True)
+            return timeout(timeout_time, use_signals=use_signals)(func)(*args, **kwargs)
```

Comment:
why we need this?

Reply:
Without this `async` tests are failing, as `timeout()` function here by default uses signals, and apparently signals can be used 'only in main thread'. So this is mainly a flag for fixing async workflows tests.